utils/utils.go

```diff
@@ -10,6 +10,10 @@ import (
 	"strings"
 )
 
+type Schema map[string]interface{}
+
+type Schemas map[string]Schema
```

Comment:
These are unnecessary. Simply use the types as is where they are needed.
If you were adding methods onto these types, then it would make sense to pull them into a util package.

Reply:
done